x-pack/platform/plugins/shared/cases/server/index.ts

```diff
@@ -16,6 +16,7 @@ export const config: PluginConfigDescriptor<ConfigType> = {
     markdownPlugins: true,
     files: { maxSize: true, allowedMimeTypes: true },
     stack: { enabled: true },
+    unsafe: { enableCaseSummary: true },
```

Comment:
@benakansara The default value for the `enableCaseSummary` is false, and here you set it to true.

```
enableCaseSummary: schema.boolean({ defaultValue: false }),
```

So with current setup is it enabled by default or not? I am confused.

Reply:
Nice catch, I'm wondering about that as well ;)

Reply:
It's not enabled by default. In order to expose this config on browser side, we need to set it to true here.

Reply:
Ok I wanted to double check that everything is setup correctly.